actionpack/test/dispatch/server_timing_test.rb

```diff
@@ -128,10 +134,6 @@ def with_test_route_set
           post "/", to: ::ServerTimingTest::TestController.action(:create)
         end
 
-        @app = self.class.build_app(set) do |middleware|
-          @middlewares.each { |m| middleware.use m }
-        end
```

Comment:
Some tests previously had their own way of drawing test routes and inserting middleware, which doesn't work now because `with_routing` in integration tests sets the integration session. I decided to just separate this pattern into two different methods instead of clearing the integration session manually.

Reply:
Does this mean application code is broken? Or just our internal code?

Reply:
Nope, I just had to rearrange how the tests chose to build the middleware that was the issue. This change doesn't impact public API of Rails / app code AFAICS.

Based on this gist with_routing was available at the instance level, but just didn't do anything. It is now available at the class level and the instance level, and actually works now.